src/Viz.js

```diff
@@ -249,6 +250,18 @@ export default class Viz extends BaseClass {
       strokeWidth: constant(0)
     };
     this._solo = [];
+
+    this._subtitleClass = new TextBox();
+    this._subtitleConfig = {
+      ariaHidden: true,
+      fontSize: 12,
+      padding: 5,
+      resize: false,
+      textAnchor: "middle"
+    };
+    this._subtitlePadding = defaultPadding;
+    this._subtitleClass = new TextBox();
```

Comment:
dubious duplicate declaration! (`this._subtitleClass` is defined twice)

Reply:
My bad 🤦‍♂️!